addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/SpiderJob.java

```diff
@@ -228,22 +208,11 @@ public void runJob(
         progress.info(
                 Constant.messages.getString(
                         "automation.info.urlsfound", this.getType(), numUrlsFound));
-        if (numUrlsFound < this.failIfFoundUrlsLessThan) {
-            progress.error(
-                    Constant.messages.getString(
-                            "automation.error.urlsfound",
-                            this.getType(),
-                            numUrlsFound,
-                            this.failIfFoundUrlsLessThan));
-        }
-        if (numUrlsFound < this.warnIfFoundUrlsLessThan) {
-            progress.warn(
-                    Constant.messages.getString(
-                            "automation.error.urlsfound",
-                            this.getType(),
-                            numUrlsFound,
-                            this.failIfFoundUrlsLessThan));
-        }
+        Control.getSingleton()
+                .getExtensionLoader()
+                .getExtension(ExtensionStats.class)
+                .getInMemoryStats()
+                .counterInc("spider.urls.added", numUrlsFound);
```

Comment:
Ideally we'll add full stats to the core spider. In which case it might be better to associate this stats name with the automation job so that the distinction is clear, eg something like `automation.spider.urls.added`?